paper-slider.html

```diff
@@ -389,6 +389,14 @@
         },
 
         /**
+         * If set, a this text will be used in the pin instead of just the numeric
+         * number. Use if you need to format the label in any way.
+         */
+        pinLabel: {
+          type: String
```

Comment:
My mistake - you do need a default value of `''` for `pinLabel` because the `value$="[[_getPinLabel(pinLabel, immediateValue)]]"` computed binding won't be evaluated unless both arguments are defined.